crates/thread_aware_cache/CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Changelog
```

Comment:
We auto-generate the changelog, so you can leave this blank for starters and it'll be filled in when releasing the crate.